src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -7373,28 +7373,7 @@ static void Main(string[] args)
         }
     }
 }";
-            var expected = @"class Test
-{
-    void method()
-    {
-        static void Main(string[] args)
-        {
-            int v = 0;
-            for(int i=0 ; i<5; i++)
-            {
-                v = NewMethod(v, i);
```

Comment:
@agocke this may simply be a case of users not terminating an existing method.  i.e. instead of:

```c#
void Foo()
{
}
void Bar()
{
}
```

they have:

```c#
void Foo()
{

void Bar()
{
}
```

Extract method should not be failing inside 'Bar'.